src/components/pager/__internal__/pager-navigation.component.js

```diff
@@ -124,7 +123,7 @@ const PagerNavigation = ({
             />
           </Label>
           <StyledPagerNoSelect>
-            {I18n.t("pager.of_y", { defaultValue: " of " })}
+            {l.pager.ofY}
```

Comment:
`ofY` should return the full string instead of the component concatenating the number.
```suggestion
            {l.pager.ofY(pageCount)}
```

Reply:
changed, thanks